test/Fhir.Metrics.Tests/Tests/TestMetrics.cs

```diff
@@ -87,8 +87,37 @@ public void BugfixAvoidLoop()
             // Is now replaced with: (((?[./])?(?[^./]+)))?
             string unit = "ForinterpretationofeGFRseehttp://www.kidney.org.au";
             Assert.Throws<ArgumentException>(() => system.Metric(unit));
-            
-            
+        }
+
+        [Fact]
+        public void MetricWithValidAnnotations()
+        {
+            Metric metric;
```

Comment:
Would be nice to have unittests that show how the expression is being rewritten, also for documentation.
Could you add extra Asserts for that?